waypoints:
  - waypoint_id: registed_0_1252124000000
  - waypoint_id: registed_0_1313938000000